source/_components/binary_sensor.envisalink.markdown

```diff
@@ -14,5 +14,26 @@ ha_iot_class: "Local Push"
 ---
 
 The `envisalink` platform allows you to get data from your [Envisalink](http://www.eyezon.com/) binary sensors from within Home Assistant.
-  
+
 The requirement is that you have setup your [Envisalink hub](/components/envisalink/).
+
+#### Sensor Types
```

Comment:
Ok, I missed that documentation somehow. What're your thoughts on removing the Envisalink binary sensor docs and directly linking there instead? There doesn't appear to be anything specialized about the Envisalink binary sensor that the general component docs don't cover. The `zones` area can include the link/documentation to the sensor type similar to how the CLI binary sensor does it.